src/engraving/libmscore/score.h

```diff
@@ -515,24 +515,14 @@ class Score : public QObject, public ScoreElement
     mu::engraving::Layout m_layout;
     mu::engraving::LayoutOptions m_layoutOptions;
 
-    ChordRest* nextMeasure(ChordRest* element, bool selectBehavior = false, bool mmRest = false);
-    ChordRest* prevMeasure(ChordRest* element, bool mmRest = false);
-    void cmdResetAllStyle();
-
     Note* getSelectedNote();
-    ChordRest* upStaff(ChordRest* cr);
-    ChordRest* downStaff(ChordRest* cr);
```

Comment:
A number of these needed to be made public in order to replace `cmd()` and `selectMove()`